pkg/config/config.go

```diff
@@ -105,3 +107,8 @@ func CacheTime() int {
 func Source() *source {
 	return &getConfig().source
 }
+
+// ConcurrentImageProcessingEnabled returns true if we want to process image using multiple cores (checking isOpaque)
+func ConcurrentImageProcessingEnabled() bool {
```

Comment:
I think we should have some other name, something like `ConcurrentOpacityCheckEnabled`? `ConcurrentImageProcessingEnabled` seems too generic.

Reply:
yes I guess that makes sense